docs/development.rst

```diff
@@ -44,6 +45,11 @@ Quickstart
   code. To run the checks without committing use ``pre-commit run``. To bypass
   the checks (to save or get feedback on work-in-progress) use ``git commit
   --no-verify``
+- If you have modified C code then run this to make it conform to the project style::
+
+  $ sudo apt-get install clang-format-7
```

Comment:
Do we absolutely need clang-format 7? I was hoping that anything >= 7 would work, and choosing an old version would 'just work' for the vast majority of people.

Reply:
I've stuck with specifying it as `sudo apt-get install clang-format` on my latest "mint" install gives version 6! Even though you can `sudo apt-get install clang-format-9`.  No idea why the default is stuck at 6.

Reply:
Jeez, that's lame. Can we target >= 6 then, or are there some options that are different? I just ran version 6 on my mint machine and I didn't notice any issues (didn't run it on all the code though).

Reply:
I'm just thinking of OSX users here --- they might think they absolutely have to have version 7 and it could be a major headache to to track down version 7 on whatever bolted on excuse for a software manager they happen to be using.

Reply:
Looks like >= 6 is ok. I had gone with 7 as it was on debian stable, but lets take pity on the OSX developer.